lib/gui.py

```diff
@@ -96,18 +96,101 @@ def on_submit():
     submit_button.grid(row=3, column=1)
 
 
+def make_port_scan(main_window):
+    font = ""
+    width = 800
+    height = 600
+    port_scan_window = tk.Toplevel(main_window, bg="#222", width=width, height=height)
```

Comment:
this `#222` is repeated constantly in other parts of the code. Why don't we make it a constant in the global scope? It would also make it easily configurable (by devs not users).

Reply:
is now set as global default_bg_color